snapcraft/plugins/python.py

```diff
@@ -205,7 +205,10 @@ def _install_pip(self, download):
 
         pip_command = [self._get_python_command(), '-m', 'pip']
 
-        # if python_command it is not from stage we don't have pip
+        # If python_command it is not from stage we don't have pip, which means
+        # we are going to need to resort to the pip installed on the system
+        # that came from build-packages. This shouldn't be a problem as
+        # stage-packages and build-packages should match.
```

Comment:
:+1: thank you. Though the typo is still there: s/python_command it is not/python_command is not/